docs/source/using-discovery/slurm.md

```diff
@@ -0,0 +1,769 @@
+
+### Using sinfo
+The `sinfo` command will show information about all partitions in the cluster, including the partition name, available nodes, and status. By default, `sinfo` reports:
+
+:::{code} bash
+| PARTITION | The list of the cluster’s partitions; a set of compute nodes grouped logically |
+| --- | --- |
+| AVAIL | The active state of the partition (up, down, idle) |
+| TIMELIMIT | The maximum job execution wall-time per partition |
+| NODES | The total number of nodes per partition |
+| STATE | See STATE table below |
+| NODELIST(REASON) | The list of nodes per partition |
+:::
+
+### Examples using sinfo
+View information about all partitions:
+
+:::{code} bash
+sinfo -a
+:::
+
+Or, a specific partition, which gives all the nodes and the states the nodes are in at the current time:
+
+:::{code} bash
+sinfo -p gpu
+PARTITION AVAIL  TIMELIMIT  NODES  STATE NODELIST
+gpu          up    8:00:00      5 drain* c[2171,2184,2188],d[1008,1014]
+gpu          up    8:00:00      3  down* c2162,d[1006,1017]
+gpu          up    8:00:00      1  drain d1025
+gpu          up    8:00:00      2   resv c2177,d1029
+gpu          up    8:00:00     50    mix c[2160,2163-2170,2172-2176,2178-2179,2185-2187,2189-2195,2204-2207],d[1001,1003-1005,1007,1009-1013,1016,1018,1020-1024,1026-1028]
+gpu          up    8:00:00      3  alloc d[1002,1015,1019]
+gpu          up    8:00:00      4   idle c[2180-2183]
+The current `TimeLimit` for the queues:
+sinfo  -o "%12P %.10A %.11l"
+PARTITION    NODES(A/I)   TIMELIMIT
+debug           402/174       20:00
+express         403/180     1:00:00
+short*          401/178  1-00:00:00
+long             224/47  5-00:00:00
+large           376/172     6:00:00
+gpu               41/17     8:00:00
+multigpu          41/17  1-00:00:00
+lowpriority     118/102  1-00:00:00
+reservation     617/402 100-00:00:0
+ai-jumpstart       2/15  2-00:00:00
+allshouse           5/7    infinite
+bansil             15/4 30-00:00:00
+ce-mri             3/10 30-00:00:00
+chen               0/12 30-00:00:00
+ctbp               0/20 30-00:00:00
+.
+.
+.
+:::
+
+View information about a specific partition (e.g., `short`, `gpu`, `long`):
+
+:::{code} bash
+sinfo -p <partition_name>
+:::
+
+Or, only view nodes in a certain state:
+
+:::{code} bash
+sinfo -p <partition> -t <state>
+:::
+
+You can use the `--Format` flag to get more information or a specific format for the output:
+
+:::{code} bash
+sinfo -p <partition> -t idle --Format=gres,nodes
+:::
+
+Below command will show detailed information about all nodes in the cluster, including the node name, state, CPU architecture, memory, and available features:
+
+:::{code} bash
+sinfo -N -l
+:::
+
+View what features a node has:
+
+:::{code} bash
+sinfo -n <node> --Format=nodes,nodelist,statecompact,features
+View what nodes are in what state in a partition using `statecompact`:
+sinfo -p <partition> --Format=time,nodes,statecompact,features,memory,cpus,nodelist
+:::
+
```

Comment:
I really like this section for empowering the user. I have found the command seff <jobid> to be super useful for scaling up jobs to use resources efficiently (or troubleshoot why a job failed). Could be a good to add somewhere here.

Reply:
Fixed bug on line 497 which was mistakenly in the code block (thanks for calling this out, as it had me notice that, too)

Reply:
@visionjo For example you could include the text below:

# View a jobs resource usage
:::{code} bash
seff <slurm-job-id>
:::

As an example of the output:
Job ID: 37320106
Cluster: discovery
User/Group: s.caplins/users
State: RUNNING
Cores: 1
CPU Utilized: 00:00:00
CPU Efficiency: 0.00% of 03:31:53 core-walltime
Job Wall-clock time: 03:31:53
Memory Utilized: 0.00 MB (estimated maximum)
Memory Efficiency: 0.00% of 1.95 GB (1.95 GB/core)
WARNING: Efficiency statistics may be misleading for RUNNING jobs.

Reply:
The example is just from an interactive srun session so not the best example.

Reply:
@SerenaCaplins, I am quite confused here.

Can you either edit directly or specify what our intent is here?